package.json

```diff
@@ -413,27 +413,14 @@
                 },
                 "default": {}
               },
-              "dataEditor.omegaEditPort": {
-                "type": "integer",
-                "description": "Editor server default port",
-                "default": 9000
-              },
-              "dataEditor.logFile": {
-                "type": "string",
-                "description": "Path to log file for data editor",
-                "default": "${workspaceFolder}/dataEditor-${omegaEditPort}.log"
-              },
-              "dataEditor.logLevel": {
-                "type": "string",
-                "description": "Log level for data editor",
-                "enum": [
-                  "error",
-                  "warn",
-                  "info",
-                  "debug",
-                  "trace"
-                ],
-                "default": "info"
+              "dataEditorConfig": {
+                "type": "object",
+                "description": "Configuration for Data Editor. Settings are omegaEditPort, logFile and logLevel",
+                "default": {
+                  "omegaEditPort": 9000,
```

Comment:
```suggestion
                  "port": 9000,
```
Maybe just `port`?

Reply:
Since it's namespaced, I think just `port` is a fine choice.